FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/appsettings.json

```diff
@@ -5,10 +5,17 @@
     "authentication": "None"
   },
   "dataLakeStore": {
-    "storageUrl": ""
+    "storageUrl": "UseDevelopmentStorage=true"
```

Comment:
Retrieve this emulator connection string?

Reply:
Remove it.